### PR TITLE
Fixed issue #80

### LINKED ISSUE #80: Apply `<AbstractAsyncAction/>` to buttons where loading is required.
Wrap "content" of buttons which perform API calls. (Note: do not wrap entire button itself)
Refer: https://github.com/profilecity/vidur/commit/3f8481f6e82f1538d63309f51e62bb700e18069c (perform changes in this manner)

```
8 results - 6 files

src/components/admin/integration/hooks/Card.vue:
  45:                 <button class="w-10 h-10 border rounded-xl border-zinc-200 hover:bg-zinc-100 hover:border-zinc-300 flex justify-center items-center" @click="open">

src/components/admin/integration/hooks/Form.vue:
  98:       <button class="flex btn btn-sm mt-4 btn-primary" :disabled="isSubmitting" type="submit">

src/components/admin/settings/general/Update.vue:
  152:             <button class="btn btn-primary" @click="onSubmit" :disabled="isSubmitting">
  192:             <button class="btn btn-primary" @click="onSubmit" :disabled="isSubmitting">

src/components/admin/settings/members/AddAction.vue:
  58:             <button class="btn btn-sm border text-sm" @click="submit(user.id, close)" :disabled="isSubmitting">Add</button>

src/components/admin/settings/members/Card.vue:
  31:           <button class="btn btn-sm border border-zinc-200 hover:border-zinc-300 flex space-x-1 items-center"

src/pages/postings/[id].vue:
   84:               <button class="btn w-full btn-primary" @click="apply" :disabled="isApplying"
  123:               <button class="btn w-full btn-primary" @click="apply" :disabled="isApplying"

```

### PR DESCRIPTION
**Pull request by ameyagidh**
Wrap "content" of buttons which perform API calls. (Note: do not wrap entire button itself)
Refer: https://github.com/profilecity/vidur/commit/3f8481f6e82f1538d63309f51e62bb700e18069c (perform changes in this manner)

8 results - 6 files

src/components/admin/integration/hooks/Card.vue:
  45:                 <button class="w-10 h-10 border rounded-xl border-zinc-200 hover:bg-zinc-100 hover:border-zinc-300 flex justify-center items-center" @click="open">

src/components/admin/integration/hooks/Form.vue:
  98:       <button class="flex btn btn-sm mt-4 btn-primary" :disabled="isSubmitting" type="submit">

src/components/admin/settings/general/Update.vue:
  152:             <button class="btn btn-primary" @click="onSubmit" :disabled="isSubmitting">
  192:             <button class="btn btn-primary" @click="onSubmit" :disabled="isSubmitting">

src/components/admin/settings/members/AddAction.vue:
  58:             <button class="btn btn-sm border text-sm" @click="submit(user.id, close)" :disabled="isSubmitting">Add</button>

src/components/admin/settings/members/Card.vue:
  31:           <button class="btn btn-sm border border-zinc-200 hover:border-zinc-300 flex space-x-1 items-center"

src/pages/postings/[id].vue:
   84:               <button class="btn w-full btn-primary" @click="apply" :disabled="isApplying"
  123:               <button class="btn w-full btn-primary" @click="apply" :disabled="isApplying"
  
  
  
  Issue FIxed #80

### CODE REVIEW

File: src/components/admin/settings/general/Update.vue [hunk 1]
```diff
@@ -1,79 +1,5 @@
...
-const isSubmitting = ref(false);
-const onSubmit = handleSubmit(async values => {
-  try {
-    isSubmitting.value = true;
-    await $fetch('/api/settings/general', {
-      method: 'PUT',
-      body: { ...values },
-    });
-
-    emits('saved');
```
Comment (amandesai01) [inline src/components/admin/settings/general/Update.vue:66]: you are changing unnecessary things, review your changes first. This is coming from not rebasing the branch.

**Comment by ameyagidh** [comment]
fixed code clean up

**Comment by amandesai01** [comment]
There are still formatting changes. Nevermind. Anyways we updating code styling so deffering this issue till then. Closing this PR and will ping you if required in future.

**Comment by ameyagidh** [comment]
Sure

**Pull request closed without merging**